linq4j/src/main/java/org/apache/calcite/linq4j/EnumerableDefaults.java

```diff
@@ -1154,11 +1154,30 @@ public void close() {
    * Returns elements of {@code outer} for which there is a member of
    * {@code inner} with a matching key. A specified
    * {@code EqualityComparer<TSource>} is used to compare keys.
+   *
+   * @deprecated Use {@link #correlateJoin(JoinType, Enumerable, Function1, Function2)}
    */
+  @Deprecated // to be removed before 2.0
   public static <TSource, TInner, TResult> Enumerable<TResult> correlateJoin(
-      final CorrelateJoinType joinType, final Enumerable<TSource> outer,
+      final CorrelateJoinType correlateJoinType, final Enumerable<TSource> outer,
       final Function1<TSource, Enumerable<TInner>> inner,
       final Function2<TSource, TInner, TResult> resultSelector) {
+    return correlateJoin(correlateJoinType.toJoinType(), outer, inner, resultSelector);
+  }
+
+  /**
+   * Returns elements of {@code outer} for which there is a member of
+   * {@code inner} with a matching key. A specified
+   * {@code EqualityComparer<TSource>} is used to compare keys.
```

Comment:
I don't see the link with `EqualityComparer<TSource>`.

Reply:
Oops, copy-paste mistake. I'll remove it from here (and from the deprecated method, where I copied it from)